client/app/components/blueprints/blueprint-details-modal/blueprint-details-modal-service.factory.js

```diff
@@ -81,20 +81,28 @@
     vm.blueprint = blueprint;
 
     vm.tabs = [
-      {"title": __('General'), "active": true},
+      {"title": __('General'), "active": false},
       {"title": __('Publish'), "active": false},
       {"title": __('Provision Order'), "active": false},
       {"title": __('Action Order'), "active": false},
     ];
 
+    if (vm.blueprint.read_only) {
```

Comment:
Action not required: consider consider using ternary operator to for conciseness 🌮